admin/src/Components/ProductForm.jsx

```
@@ -1,86 +1,69 @@
+const ProductForm = ({ product, onDelete, onSave }) => {
+    const [actions, setActions] = useState([]);
+    const [availableActionTypes, setAvailableActionTypes] = useState([]);
+    const [selectedActionType, setSelectedActionType] = useState(null);
+    const [saveDisabled, setSaveDisabled] = useState(true);
+
+    useEffect(() => {
+        const handleProductChange = () => {
+            const newActions = product.actions;
+            const newAvailableActionTypes = filterAvailableActions(newActions);
+            const selectedActionType = filterSelectedActionType(
+                selectedActionType,
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)